save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/contests/ContestSampleListCard.kt

```diff
@@ -0,0 +1,76 @@
+@file:Suppress(
+    "HEADER_MISSING_IN_NON_SINGLE_CLASS_FILE",
+)
+
+package com.saveourtool.save.frontend.components.views.contests
+
+import com.saveourtool.save.entities.contest.ContestSampleDto
+import com.saveourtool.save.frontend.components.tables.*
+import com.saveourtool.save.frontend.externals.fontawesome.faCode
+import com.saveourtool.save.frontend.utils.*
+import react.FC
+import react.Fragment
+import react.VFC
+import react.create
+import react.dom.html.ReactHTML.div
+import react.dom.html.ReactHTML.td
+import react.router.dom.Link
+import web.cssom.ClassName
+
+val contestSampleList = contestSampleListCard()
+
+@Suppress(
+    "MAGIC_NUMBER",
+    "TYPE_ALIAS",
+)
+private val contestSampleTable: FC<TableProps<ContestSampleDto>> = tableComponent(
+    columns = {
+        columns {
+            column(id = "name", header = "Name", { name }) { cellContext ->
+                Fragment.create {
+                    td {
+                        Link {
+                            // FixMe: need add link to = ""
```

Comment:
?

Reply:
This will be added in the next PR. There is no view to view a specific template yet.